### PR TITLE
Bug/8162

### PR DESCRIPTION
**Pull request by karthik-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/8162

*Description of changes:*
* Display on hold reason for "On Hold" tasks on the review account component

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/staff/review-task/AccountInformation.vue [hunk 1]
```diff
@@ -75,7 +75,7 @@ export default class AccountInformation extends Vue {
 
   @Prop({ default: 'Account Information' }) private title: string
   @Prop({ default: {} }) accountUnderReview: any
-  @Prop({ default: {} }) accountUnderReviewAddress: Address
+  @Prop({ default: null }) accountUnderReviewAddress: Address
```
Comment (saravanpa-aot) [inline auth-web/src/components/auth/staff/review-task/AccountInformation.vue:78]: why this change?

Comment (karthik-aot) [reply auth-web/src/components/auth/staff/review-task/AccountInformation.vue:78]: There was this warn message  
image

**Review by saravanpa-aot — APPROVED** [review]

**Review by shabeeb-aot — APPROVED** [review]
